build/docker/centos7/build/Dockerfile

```diff
@@ -1,43 +1,65 @@
 FROM centos:7
 
 WORKDIR /tmp
-
-RUN rpmkeys --import "http://pool.sks-keyservers.net/pks/lookup?op=get&search=0x3fa7e0328081bff6a14da29aa6a19b38d3d831ef" && \
+COPY mono-project.com.rpmkey.pgp ./
+RUN rpmkeys --import mono-project.com.rpmkey.pgp && \
     curl -Ls https://download.mono-project.com/repo/centos7-stable.repo -o /etc/yum.repos.d/mono-centos7-stable.repo && \
     yum repolist && \
     yum install -y \
-        centos-release-scl \
-        scl-utils && \
+        centos-release-scl-rh \
+        epel-release \
+        scl-utils \
+        yum-utils && \
     yum install -y \
         autoconf \
         automake \
         binutils-devel \
         curl \
+        debbuild \
         devtoolset-8 \
         devtoolset-8-libubsan-devel \
         devtoolset-8-valgrind-devel \
         dos2unix \
         dpkg \
+        gettext-devel \
         git \
-        go-toolset-7 \
-        java-11-openjdk-devel.x86_64 \
+        golang \
+        java-11-openjdk-devel \
```

Comment:
This pulls in a lot of dependencies and is probably one of the main reasons for the large image... Debian has a headless package to solve this problem, but I couldn't find anything similar for CentOS. I don't think a change is needed here, just want to leave this comment here so people are aware.